Adjust docker and grype tasks (#355)
oracle/graalvm-reachability-metadata: CONTRIBUTING.md

 
### Providing the tests that use docker
 
If your tests use Docker (either with explicit Docker process invocation or through some library method call), all images have to be declared in `required-docker-images` file.
This file must be placed under `/tests/src/<groupId>/<artifactId>/<versionId>`.

> Only Docker images that are listed in [allowed-docker-images directory](https://github.com/oracle/graalvm-reachability-metadata/blob/master/tests/tck-build-logic/src/main/resources/allowed-docker-images) can be executed.

Reviewer: ```suggestion
Only Docker images that are listed in the [`allowed-docker-images` directory](https://github.com/oracle/graalvm-reachability-metadata/blob/master/tests/tck-build-logic/src/main/resources/allowed-docker-images) can be used for testing.
```

If you want to extend this list, please create separate pull request to do that.

> That pull request should add a new file in [allowed-docker-images directory](https://github.com/oracle/graalvm-reachability-metadata/blob/master/tests/tck-build-logic/src/main/resources/allowed-docker-images)

Reviewer: ```suggestion
That pull request should add a new file in the [`allowed-docker-images` directory](https://github.com/oracle/graalvm-reachability-metadata/blob/master/tests/tck-build-logic/src/main/resources/allowed-docker-images)
```

with the name in the format `Dockerfile-<dockerImageName>` (replace all occurrence of `/` with `_`) .

> The only line that this file should contain is `FROM <dockerImageName>`.

Reviewer: ```suggestion
The only line that this file needs to contain is `FROM <dockerImageName>`.
```

> Once open a pull request, please post the result of the following command in your pull request description:

Reviewer: ```suggestion
Once you have opened such a pull request, please post the result of the following command in your pull request description:
```
